Bump/jetbrains 2024.3.1.1

jamesward opened the pull request:
Bump jetbrains IDEs to latest:
```
    jetbrains: 2024.3 -> 2024.3.3
    
    jetbrains.aqua: 2024.3 -> 2024.3.1
    jetbrains.clion: 2024.3.1 -> 2024.3.1.1
    jetbrains.datagrip: 2024.3.2 -> 2024.3.3
    jetbrains.dataspell: 2024.3.1 -> 2024.3.1.1
    jetbrains.gateway: 2024.3.1 -> 2024.3.1.1
    jetbrains.idea-community: 2024.3.1 -> 2024.3.1.1
    jetbrains.idea-ultimate: 2024.3.1 -> 2024.3.1.1
    jetbrains.phpstorm: 2024.3.1 -> 2024.3.1.1
    jetbrains.pycharm-community: 2024.3.1 -> 2024.3.1.1
    jetbrains.pycharm-professional: 2024.3.1 -> 2024.3.1.1
    jetbrains.rider: 2024.3.2 -> 2024.3.3
    jetbrains.ruby-mine: 2024.3.1 -> 2024.3.1.1
    jetbrains.rust-rover: 2024.3.1 -> 2024.3.2
    jetbrains.webstorm: 2024.3.1 -> 2024.3.1.1
    jetbrains.writerside: 2024.3 EAP -> 2024.3 EAP
```

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

theCapypara commented:
Thanks!
#366290 should be merged before and then you should re-run the plugin update script to make sure plugins for source builds are also up-to-date

leona-ya commented:
https://github.com/NixOS/nixpkgs/pull/366290 is now merged. Can you please rerun the update script including the plugin update.

jamesward committed:
jetbrains: 2024.3 -> 2024.3.3

jetbrains.aqua: 2024.3 -> 2024.3.1
jetbrains.clion: 2024.3.1 -> 2024.3.1.1
jetbrains.datagrip: 2024.3.2 -> 2024.3.3
jetbrains.dataspell: 2024.3.1 -> 2024.3.1.1
jetbrains.gateway: 2024.3.1 -> 2024.3.1.1
jetbrains.idea-community: 2024.3.1 -> 2024.3.1.1
jetbrains.idea-ultimate: 2024.3.1 -> 2024.3.1.1
jetbrains.phpstorm: 2024.3.1 -> 2024.3.1.1
jetbrains.pycharm-community: 2024.3.1 -> 2024.3.1.1
jetbrains.pycharm-professional: 2024.3.1 -> 2024.3.1.1
jetbrains.rider: 2024.3.2 -> 2024.3.3
jetbrains.ruby-mine: 2024.3.1 -> 2024.3.1.1
jetbrains.rust-rover: 2024.3.1 -> 2024.3.2
jetbrains.webstorm: 2024.3.1 -> 2024.3.1.1
jetbrains.writerside: 2024.3 EAP -> 2024.3 EAP

jamesward committed:
jetbrains.plugins: update

jamesward commented:
I rebased on master and re-ran the update script. IntelliJ seems to be working as expected.

leona-ya reviewed (approved):
Looks good, thank you very much!
